.beislid/ship-it-templates.md

```
@@ -42,6 +42,14 @@ Each phase prints an entry summary on its way in and an exit summary on its way
 💭 Phase 2b: skipped at user request.
 ```
 
+Gate execution failures use the shared Gate result envelope from `output-templates.md`:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)